.kokoro/build.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
+# Copyright 2018 Google LLC
```

Comment:
Is 2018 correct? Is it because that's the creation/addition date of this file?

Also, should config files contain the license header, too?

Reply:
2018 is the date this file was originally created (this is a template file).

The Open Source folks seem to recommend license headers for config files as well. Those are all changes in the synthtool templates. I'll open a PR there.

> any files that can be added to (i.e. anything that takes the format of a source file and supports file comments). This would include things like YAML files.

Reply:
https://github.com/googleapis/synthtool/pull/458